src/components/List/List.tsx

```diff
@@ -0,0 +1,21 @@
+import type { FC, PropsWithChildren } from 'react';
+import { memo, Fragment } from 'react';
+
+interface Props {
+
+  /** List size. */
+  readonly size: number;
+}
+
+/** Component renders children * size times. */
+const ListComponent: FC<PropsWithChildren<Props>> = ({ size, children }) => {
+  const items = [...Array(size).keys()].map(i => <Fragment key={i}>{children}</Fragment>);
```

Comment:
Should it be named `List` though? It seems it just duplicate its chidlren n times